carta/image.py

```diff
@@ -477,17 +477,24 @@ def set_scaling(self, scaling, alpha=None, gamma=None, min=None, max=None):
             The alpha value (only applicable to ``LOG`` and ``POWER`` scaling types).
         gamma : {2}
             The gamma value (only applicable to the ``GAMMA`` scaling type).
-        min : {3}
-            The minimum of the scale. Only used if both *min* and *max* are set.
-        max : {4}
-            The maximum of the scale. Only used if both *min* and *max* are set.
+        clip : {3}
+            The scale clip. In percentage. Can not be used witn *min* and *max*.
+        min : {4}
+            The minimum of the scale. Only used if both *min* and *max* are set. Can not be used with *clip*.
+        max : {5}
+            The maximum of the scale. Only used if both *min* and *max* are set. Can not be used with *clip*.
         """
         self.call_action("renderConfig.setScaling", scaling)
         if scaling in (Scaling.LOG, Scaling.POWER) and alpha is not None:
             self.call_action("renderConfig.setAlpha", alpha)
         elif scaling == Scaling.GAMMA and gamma is not None:
             self.call_action("renderConfig.setGamma", gamma)
-        if min is not None and max is not None:
+        if clip is not None:
+            min = "None"
+            max = "None"
+            self.call_action("renderConfig.setPercentileRank", clip)
+        elif min is not None and max is not None:
+            clip = "None"
             self.call_action("renderConfig.setCustomScale", min, max)
```

Comment:
There is already a high-level function called `set_percentile_rank`, which changes only the rank -- it should probably be used here instead of the low-level action call.

Reply:
Thanks for the reminding of the `set_percentile_rank`, I didn't know the existence of that function. I used it in the `set_scaling` instead of action call. To avoid confusion, I thus change the argument `clip` to `rank`.